Extensions/audio_plus.css

```diff
@@ -33,36 +33,98 @@
 	border-radius: 6px;
 }
 
-.xkit-audio-plus-controls {
+#right_column.has_docked_audio {
+	opacity: 0.3;
+}
+
+.xkit-audio-plus-pseudo-post {
+	display: block;
 	position: fixed;
-	bottom: 50px;
-	left: 50%;
-	transform: translateX(245px) translateY(0px);
+	background: white;
+	height: auto;
+	width: 300px;
+	border-radius: 3px;
+	padding: 15px 0;
+	left: calc(50% + 178px);
+	bottom: 15px;
+	opacity: 0;
+	z-index: -99999;
+	transition:
+		bottom .15s cubic-bezier(.165,.84,.44,1),
+		transform .15s cubic-bezier(.165,.84,.44,1), 
+		opacity .15s cubic-bezier(.165,.84,.44,1);
+}
 
-	z-index: 10000;
-	width: 85px;
+#search_actions_search .xkit-audio-plus-pseudo-post {
+	left: calc(50% + 289px);
+}
+
+.xkit-audio-plus-pseudo-post.showing {
+	bottom: 20px;
+	opacity: 1;
+	z-index: 99999;
```

Comment:
`78` would be good here (the video dock is 79 and i think the audio popout being obscured for a second is better than the video being obscured while it docks)